components/automate-chef-io/content/docs/reports.md

```diff
@@ -11,12 +11,10 @@ toc = true
     weight = 10
 +++
 
-## Overview
-
-The Reports view under the Compliance tab provides comprehensive insight into the compliance status of all scanned infrastructure.
+The _Reports_ page under the _Compliance_ dashboard provides comprehensive insight into the compliance status of all scanned infrastructure.
```

Comment:
```suggestion
The _Reports_ page under the _Compliance_ ( Compliance > Reports) dashboard provides comprehensive insight into the compliance status of all scanned infrastructure.
```
What do you think of adding this sort of breadcrumb in the docs?

Reply:
I love this idea!